modules/rules_detekt/0.8.1.1/presubmit.yml

```diff
@@ -0,0 +1,15 @@
+matrix:
+  platform: [ "macos", "ubuntu2004" ]
+
+tasks:
+  verify_targets:
+    name: "Verify build targets"
+    platform: ${{ platform }}
+    build_flags:
+      - "--enable_bzlmod=true"
+    test_flags:
+      - "--enable_bzlmod=true"
+    build_targets:
+      - "@rules_detekt//..."
```

Comment:
Dev dependencies aren't available in CI when you build a module, so stardoc targets in `//docs` fail. If you want dev deps to be available, you need to define a [test module](https://github.com/bazelbuild/bazel-central-registry/blob/main/docs/README.md#test-module).

Reply:
Yeah just narrowed the build/test scope down some to get the pre-submit passing. https://github.com/buildfoundation/bazel_rules_detekt/commit/c0e2e4e55bb9f04a8fbc444f5e015a535dbbdeca